apps/site/package.json

```diff
@@ -2,30 +2,31 @@
   "name": "@node-core/website",
   "type": "module",
   "scripts": {
-    "prebuild": "pnpm build:blog-data",
-    "build": "pnpm build:default --turbo",
-    "build:default": "cross-env NODE_NO_WARNINGS=1 next build",
+    "build": "node --run build:default -- --turbo",
     "build:blog-data": "node ./scripts/blog-data/generate.mjs",
     "build:blog-data:watch": "node --watch --watch-path=pages/en/blog ./scripts/blog-data/generate.mjs",
-    "check-types": "tsc --noEmit",
-    "deploy": "cross-env NEXT_PUBLIC_STATIC_EXPORT=true pnpm build:default",
+    "build:default": "cross-env NODE_NO_WARNINGS=1 next build",
+    "cloudflare:build:worker": "opennextjs-cloudflare build",
+    "cloudflare:deploy": "wrangler deploy",
+    "cloudflare:preview": "wrangler dev",
+    "deploy": "cross-env NEXT_PUBLIC_STATIC_EXPORT=true node --run build:default",
     "dev": "cross-env NODE_NO_WARNINGS=1 next dev --turbo",
     "lint": "turbo run lint:md lint:js lint:css",
     "lint:css": "stylelint \"**/*.css\" --allow-empty-input --cache --cache-strategy=content --cache-location=.stylelintcache",
     "lint:fix": "turbo run lint:md lint:js lint:css --no-cache -- --fix",
     "lint:js": "eslint \"**/*.{js,mjs,ts,tsx}\"",
     "lint:md": "eslint \"**/*.md?(x)\" --cache --cache-strategy=content --cache-location=.eslintmdcache",
+    "lint:types": "tsc --noEmit",
     "playwright": "playwright test",
+    "prebuild": "node --run build:blog-data",
+    "predeploy": "node --run build:blog-data",
     "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "serve": "pnpm dev",
+    "serve": "node --run dev",
     "start": "cross-env NODE_NO_WARNINGS=1 next start",
     "sync-orama": "node ./scripts/orama-search/sync-orama-cloud.mjs",
-    "test": "turbo test:unit",
+    "test": "test:unit",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 node --experimental-test-coverage --test-coverage-exclude=**/*.test.* --experimental-test-module-mocks --enable-source-maps --import=global-jsdom/register --import=tsx --import=tests/setup.jsx --test **/*.test.*",
-    "test:unit:watch": "cross-env NODE_OPTIONS=\"--watch\" pnpm test:unit",
-    "cloudflare:build:worker": "opennextjs-cloudflare build",
-    "cloudflare:deploy": "wrangler deploy",
-    "cloudflare:preview": "wrangler dev"
+    "test:unit:watch": "node --run test:unit -- --watch"
```

Comment:
Does this work, or does --watch get passed as a file? The old one used NODE_OPTIONS intentionally

Reply:
for more info it's because last arg after a glob is considered as a second blog on test runner

Reply:
So the command before was wrong actually. `--watch` is an argument for Node's test runner. But `--watch _can also_ be an argument for `--run` which tells it to rerun it if files change. So it is confusing, as we want to pass `--watch` not to `--run` but to the `test runner`